nf_core/lint/__init__.py

```diff
@@ -205,7 +205,7 @@ def _lint_pipeline(self):
                 "Running lint checks", total=len(self.lint_tests), func_name=self.lint_tests[0]
             )
             for fun_name in self.lint_tests:
-                if self.lint_config.get(fun_name) is False:
+                if self.lint_config.get(fun_name, True) is False:
```

Comment:
This way it's true by default, and only skipps if there's an actual `False` in the config.

Reply:
It's fine as it was before I think - if it's not set in the config it will default to `None`, which is not `False`. So already only skips if there's a `False` in the config..

Reply:
Problem was more this:
`if fn in self.lint_config.get("files_exist", []):`
this just skips if the condition evaluates to true.

It also gave me a "boolean not iterable" error or something, probably because of `in`
No problem in changing it back to the empty list instead of 'True' if you like that better, however the stuff with the `in` didn't work for me.

Reply:
I'm a bit confused.. This config:

```yaml
files_exist:
  - 'environment.yml'
  - 'Dockerfile'
```

And the above code - the entire test should run, right?

Then separately, in `lint/files_exist.py` you have the code `if fn in self.lint_config.get("files_exist", []):` and that throws an error?

I can imagine that it might fail if you set `files_exist: true` in the config. But you shouldn't need to ever set that as to run it you just omit it from the config. Then if it's `files_exist: false` then the entire test is skipped so it never gets to the loop code.

Reply:
Trying again :)

Reply:
Ah okay no we were talking about different files :D 
I was talking about `__init__.py`
I think there we agree it's just a question whether we have `True`or `{}`in the get-function - obviously doesn't really matter :) And in `lint/files_exist.py` I just get the files that should be ignored and then check with an if-statement.

Speaking of which - was wondering whether you want logging there, that we ignore these files?

Reply:
I think that they are functionally equivalent, so I don't really mind. I would generally prefer the shorter one but if you think that the `True` default is clearer then that's fine too (and I guess that has the benefit that people _can_ write `testname: true` in their yaml if they really want).

Logging - please add a message to `results['ignored']` for each file that is skipped. The downstream reporting of this should already be in place.

Reply:
Great, will do!